client/components/payment-activity/index.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { Card, CardBody, CardHeader } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies.
+ */
+import './style.scss';
+
+const PaymentActivity: React.FC = () => {
+	return (
+		<Card className="wcpay-payments-activity__card">
+			<CardHeader
+				className="wcpay-payments-activity__card__header"
+				isBorderless={ true }
+			>
```

Comment:
💭 There may be an opportunity to reduce the amount of custom css applied here. 

I attempted the following and see no major difference with the rendered component (yet, I know there may be more to come, but we can let that be implemented as-needed). 

```suggestion
		<Card>
			<CardHeader>
```

Reply:
Thanks, @Jinksi. I have added these classes since: 

1. I found that the Card by default does not have any padding and the inner individual constituents have their own padding. I thought it is better to add the padding to the whole wrapper as per our design, for uniformity.  This will also help have same padding for upcoming components.

2. The CSS on header card removes its own padding and there is an `isBorderless` prop set to `true` to remove bottom border on the header. 

I hope this makes sense.

Reply:
Agree with @Jinksi here – not so much "let's avoid css classes", more that we should avoid overriding / customising styles. 

If the components aren't doing what we want (i.e. differ from design in subtle or major ways), we should:

- adapt the design to match the components - i.e. we can implement the intended design but it looks a little different (though likely more consistent with rest of app)
- or 
- adapt the components so they have the flexibility we need - i.e. this design proposes changes or additions to components, and we need to implement in component

Reply:
Thanks for sharing some very helpful guidance. I removed the custom css, and found the only difference it made was adding some extra 8 px padding. I also now see that has been removed in the new simplified design update, making it consistent with the headings of other widgets.

Reply:
Since I had already merged this PR before @haszari 's comment, i have created a fresh PR with the tweak 

https://github.com/Automattic/woocommerce-payments/pull/8507